### DISCUSSION TITLE
v1.9.0添加pr master分支login页面不正常

### DISCUSSION THREAD

**Discussion by bardyang** [hhyo/Archery#1822] [Q&A]
![image](https://user-images.githubusercontent.com/13637514/188808980-e664df74-d4ed-4cfe-95fa-3b2120676aab.png)
显示的页面内容用户名和密码不是标准居中的样式

**Comment by nick2wang** [comment]
静态资源加载失败，请检查是否开启debug模式以及static路径配置
